.github/workflows/build-server.yml

```diff
@@ -1,16 +1,26 @@
 name: Build+Test Server
 
-on: [ push, pull_request ]
+on:
+  push:
+    paths:
+      - "server/**/*"
+  pull_request:
+    paths:
+      - "server/**/*"
 
 jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
-    - name: Use Node.js 18
-      uses: actions/setup-node@v1
-      with:
-        node-version: 18
-    - run: cd server && yarn
-    - run: cd server && yarn build
-    - run: cd server && yarn test
+      - uses: actions/checkout@v3
+      - name: Use latest Node.js LTS
+        uses: actions/setup-node@v3
+        with:
+          node-version: "lts"
```

Comment:
Probably better to stick with a defined version

Reply:
We could test with both an old LTS version (for existing installs) and the latest node version.

Reply:
How about both lts and latest?